### PR TITLE
[REVIEW] Updated Docs

### LINKED ISSUE #586: [BUG] Documentation need updating
**Describe the bug**
[https://github.com/rapidsai/cugraph/blob/branch-0.11/CONTRIBUTING.md]
under "Setting Up Your Build Environment" it does not list Cudf as a dependency.  This might seem obvious to the developers, but it can be confusing to Noobs.

### PR DESCRIPTION
**Pull request by BradReesWork**
Updated docs

closes #586

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ The [RAPIDS](https://rapids.ai) cuGraph library is a collection of graph analyti
 import cugraph
 
 # assuming that data has been loaded into a cuDF (using read_csv) Dataframe
+gdf = cudf.read_csv("graph_data.csv", names=["src", "dst"], dtype=["int32", "int32"] )
+
 # create a Graph using the source (src) and destination (dst) vertex pairs the GDF  
 G = cugraph.Graph()
 G.add_edge_list(gdf["src"], gdf["dst"])
```
Comment (afender) [inline README.md]: We may want to use the new 0.11 API here
```
    G = cugraph.DiGraph()
    G.from_cudf_edgelist(gdf, source='src', destination='dst')
```

File: README.md [hunk 2]
```diff
@@ -32,12 +34,15 @@ for i in range(len(gdf_page)):
 | :-------------------------------------------- | ---------- | ---------------------------- |
 | PageRank                                      | Multi-GPU  |                              |
 | Personal PageRank                             | Single-GPU |                              |
+| Katz Centrality                               | Single-GPU |                              |
 | Jaccard Similarity                            | Single-GPU |                              |
 | Weighted Jaccard                              | Single-GPU |                              |
 | Overlap Similarity                            | Single-GPU |                              |
 | SSSP                                          | Single-GPU | Updated to provide path info |
 | BFS                                           | Single-GPU |                              |
```
Comment (afender) [inline README.md]: Should we mention that we also have a BSP variant for BFS in the Notes?

File: README.md (hunk 1 above)
Comment (BradReesWork) [reply README.md]: I'll update that plus add a note/link to the transition page

File: README.md (hunk 2 above)
Comment (BradReesWork) [reply README.md]: done

**Comment by BradReesWork** [comment]
rerun tests

**Pull request merged**